reviewrot/githubstack.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import datetime
+from dateutil.relativedelta import relativedelta
+from basereview import BaseService, BaseReviewRot
+from github import Github
+from github.GithubException import UnknownObjectException
+
+log = logging.getLogger(__name__)
+
+
+class GithubService(BaseService):
+    """
+    This class represents Github. The reference can be found here:
+    https://developer.github.com/v3/
+    """
+    def request_reviews(self, user_name, repo_name=None, state_=None,
+                        value=None, duration=None, token=None, host=None):
+        """
+        Creates a github object.
+        Requests pull requests for specified username and repo name.
+        If repo name is not provided then requests pull requests
+        for all repos for specified username/organization.
+
+        Args:
+            user_name (str): Github username or organization name
+            repo_name (str): Github repository name for specified
+                             username or organization
+            state_ (str): The filter state for pull requests, e.g, older
+                          or newer
+            value (int): The value in terms of duration for requests
+                         to be older or newer than
+            duration (str): The duration in terms of period(year, month, hour,
+                            minute) for requests to be older or newer than
+            token (str): Github token for authentication
+            host (str): Github host name (This value is not yet supported.
+                        Default behavior is to use public github instance.)
+        """
+        # get authenticated github object
+        g = Github(token)
+        log.debug('Github instance created: %s', g)
+        try:
+            # get user object
+            uname = g.get_user(user_name)
+        except UnknownObjectException as e:
+            log.debug('Invalid username/organizaton: %s', user_name)
+            log.debug(e)
```

Comment:
There is a logger method which automatically includes details about an exception and it assigns the error level. Any reason you can't use this instead of these two debug logs?

```python
log.exception('Invalid username/organization: %s', user_name)
```